.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Kindly create a test.md file and ignore that file here As part of the project requirement.